addons/l10n_uy/data/template/account.account-uy.csv

```diff
@@ -71,8 +72,9 @@
 "uy_code_21320","Other debts","21320","liability_current","False","Otras deudas"
 "uy_code_21401","Minimum Sales Tax","21401","liability_current","False","Iva Ventas Mínima"
 "uy_code_21402","Basic Sales Tax","21402","liability_current","False","Iva Ventas Básica"
-"uy_code_21403","Tax to be Paid","21403","liability_payable","True","Iva a Pagar"
-"uy_code_21404","Withholding Tax","21404","liability_current","False","Iva Retenido"
+"uy_code_21403","Exempt Sales Tax","21403","liability_current","False","Iva Ventas Exento"
+"uy_code_21404","Tax to be Paid","21404","liability_payable","True","Iva a Pagar"
+"uy_code_21405","Withholding Tax","21405","liability_current","False","Iva Retenido"
 "uy_code_21411","Withholding tax","21411","liability_current","False","Irpf Retenido"
 "uy_code_21421","Social Security Bank of Uruguay","21421","liability_current","False","Bps"
```

Comment:
This is weird to have Withholding Tax double @zaoral

Reply:
Hi @jco-odoo 
Indeed, but I think there are different account each one for a different tax.
If you go to the last column the translation one you will see "Iva Retenido" and "Irpf Retenido"

Maybe need to change the English name of the account to avoid confusion. Any suggestion @fvz-odoo @MarianaLuzTorres I think we should do the name change in this pr or in a different PR?

Best Regards

Reply:
@fvz-odoo @MarianaLuzTorres ping

Reply:
They are 2 different taxes, 2 different withholding. for me changing the english name of the account is the solution

Reply:
@fvz-odoo Something is wrong with the English translations.  Maybe IVA can be translated to VAT instead?  How can IRPF be retenido when IRPF is what is the withholding normally?  How do accountants understand themselves?

Reply:
> @fvz-odoo Something is wrong with the English translations. Maybe IVA can be translated to VAT instead? How can IRPF be retenido when IRPF is what is the withholding normally? How do accountants understand themselves?

Hi @jco-odoo 

IVA can be translated to VAT, no problem. But IRPF, I don't think I understand your question, IRPF is a type of tax, not always a withholding, there can be a withholding on the IRPF as there can be a withholding on the IVA...

Let me know if this clarifies your question...

Cheers!

Fernando

@zaoral @MarianaLuzTorres @ren-odoo

Reply:
IRPF means "Impuesto a la renta personas fisicas"